include/bringauto/logging/SizeLiterals.hpp

```diff
@@ -0,0 +1,25 @@
+/**
+ * @brief This file defines size literals
+ */
+#pragma once
+
+auto operator ""_B(unsigned long long const x)
+-> long { return x; }

```

Comment:
unsigned long

Reply:
2^32 is just 4 gigs of RAM, we need bigger files